org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/xerces/LSPXMLEntityResolver.java

```diff
@@ -0,0 +1,39 @@
+package org.eclipse.lemminx.extensions.xerces;

```

Comment:
Don't forget license header for new files.

Reply:
fixed